client/datatypes.go

```diff
@@ -2,9 +2,9 @@
 // User Types
 //
 // Generated with goagen v0.2.dev, command line:
-// $ goagen
+// $ goagen.exe
```

Comment:
What goes upstream != what's in the PR

We can easily have 2 commits in PR, 1 with change and 1 with regen. Then squash those before it hits the master branch.

When you refer to editing the generated code I hope you're only referring to e.g. main.go?